src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -194,6 +195,55 @@ bool CommandRequestImpl::CheckSyntax(const std::string& str,
   return true;
 }
 
+smart_objects::SmartObject CreateUnsupportedResourceResponse(
+    const hmi_apis::FunctionID::eType function_id,
+    const uint32_t hmi_correlation_id,
```

Comment:
@wolfylambova22 As far as I know this rule doesn't apply to paramters of methods.